ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/ImageEditor.kt

```diff
@@ -1,29 +1,8 @@
 
 package org.wordpress.android.imageeditor
 
-import android.content.Context
-import android.content.Intent
-import org.wordpress.android.imageeditor.fragments.MainImageFragment
+import javax.inject.Inject
+import javax.inject.Singleton
 
-class ImageEditor {
-    var imageUrls: ArrayList<String> = ArrayList()
-
-    val isSingleImageAndCapability: Boolean
-        get() = imageUrls.size == 1
-
-    /**
-     * @param context self explanatory
-     * @param contentUri URI of initial media - can be local or remote
-     */
-    fun edit(
-        context: Context,
-        contentUri: String
-    ) {
-        // TODO
-        // Temporarily goes to edit image activity
-        val intent = Intent(context, EditImageActivity::class.java)
-        intent.putExtra(MainImageFragment.ARG_MEDIA_CONTENT_URI, contentUri)
-
-        EditImageActivity.startIntent(context, intent)
-    }
-}
+@Singleton
```

Comment:
I'm not sure what exactly will we put into this class, but I'm not sure we'll want to keep it as a singleton. I guess we can keep it for now, but we need to be careful about what we add.

Reply:
Removed unused class here af5488a